packages/shared/src/components/profile/UserShortInfo.tsx

```diff
@@ -108,9 +110,18 @@ const UserShortInfoComponent = <Tag extends React.ElementType>(
             )}
             <ReputationUserBadge user={user} />
           </div>
-          <TruncateText className="text-text-secondary" title={`@${username}`}>
-            {transformUsername ? transformUsername(user) : `@${username}`}
-          </TruncateText>
+          <div className="flex text-text-secondary">
+            <TruncateText title={`@${username}`}>
+              {transformUsername ? transformUsername(user) : `@${username}`}
+            </TruncateText>
+
+            {topReader && (
+              <>
+                <Separator />
+                <TopReaderIn topReader={topReader} />
```

Comment:
We could have probably combined the Seperator inside the TopReaderIn, but it's fine as is

Reply:
Thought about it too, but considered not too at it might be confusing (DX wise) why the other elements has a separator in front, and not this one (even if it’s inside it)